article/_posts/2019-09-02-Evaluating-the-Search-Phase-of-Neural-Architecture-Search.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: review
+title: "Evaluating the Search Phase of Neural Architecture Search"
+tags: NAS deep-learning
```

Comment:
Does not seem like NAS is listed in the tags file.

Reply:
Argh I forgot to include it! Fixed in 540cfa0